client/src/routes/Workspace.css

```diff
@@ -1,5 +1,11 @@
 @import "../reset.css";
 
+html,body {
+  overflow-x: hidden;
+  height: 100%;
+  width: 100%;
+}

```

Comment:
If we use just tag names, this will be applied to every page on our app. You can probably make a div with a class name of `wrapper` and surround the inner elements with it.

Reply:
in this case though, html/body will need to be 100% anyway. I can make a wrapper for the overflow though! thanks

Reply:
Done 👍🏽